src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -1003,7 +1011,7 @@ protected override void EndProcessing()
 
                     BuildPrivateDataInModuleManifest(result, streamWriter);
 
-                    BuildModuleManifest(result, "HelpInfoURI", Modules.HelpInfoURI, !string.IsNullOrEmpty(_helpInfoUri), () => QuoteName(_helpInfoUri), streamWriter);
+                    BuildModuleManifest(result, "HelpInfoURI", Modules.HelpInfoURI, !string.IsNullOrEmpty(_helpInfoUri), () => QuoteName((null != _helpInfoUri) ? new Uri(_helpInfoUri) : null), streamWriter);
```

Comment:
Since QuoteName() takes a null but Uri constructor doesn't accept null, I think we'd end up with a bunch of code that makes it harder to read in this particular case.